docs_src/microservices/security/edgex-config.1.md

```diff
@@ -0,0 +1,25 @@
+% edgex-config(1) User Manuals edgex-config(1)
+
+# NAME
+
+**edgex-config** – Perform post-installation EdgeX configuration
+
+# SYNOPSIS
+
+**edgex-config** [OPTIONS] COMMAND [ARG...]
+
+# DESCRIPTION
+
+**edgex-config** performs post-installation EdgeX configuration, mostly regarding secrets stored in the EdgeX secret store. edgex-config takes a command that specifies which module is being configured, and module-specific arguments thereafter.
```

Comment:
We already have Consul to address all the configuration contained in the TOML, so what configuration is this meant to address? 
Would prefer the utility name be more specific since `edgex-config` sounds like it addresses service's TOML config.

Reply:
Ideas solicited.  Command interface is modeled after "docker <subcommand> <args>"

Reply:
Changed to edgex-secrets-config for now.